.github/workflows/push.yml

```diff
@@ -5,14 +5,22 @@ on:
     types: [opened, synchronize]
   merge_group:
     types: [checks_requested]
+  push:
```

Comment:
Did not know about this! Can you send a link? Does this apply to all other repos?

Reply:
I found this
https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#about-caching-workflow-dependencies
It seems to also work for multiple runs in the current branch. But no sharing between non default branches for security reasons.